fedimint-cli/src/client.rs

```diff
@@ -161,15 +161,25 @@ pub enum ClientCmd {
     },
     /// Call a module subcommand
     Module {
+        #[clap(subcommand)]
+        command: ModuleSubcommand,
         /// Module selector (either module id or module kind)
-        #[clap(long)]
-        module: ModuleSelector,
+        module: Option<ModuleSelector>,
         args: Vec<ffi::OsString>,
```

Comment:
likely subcommand and args are incompatible

Reply:
is it because of making it a postional argument ?

Reply:
no, just incompatible
args should be part of subcommand

Reply:
@Maan2003 what can be the problem now ?

Reply:
I think,  backward incompatibility test fails due to this error  ->

![image](https://github.com/fedimint/fedimint/assets/133089376/0b380874-b4e3-4834-846a-07cd8ee347d7)


Please tell how to remove it.